src/2-run-ipfs-on-a-vps/index.md

```diff
@@ -0,0 +1,79 @@
+---
+baseurl: ..
+template: tmpl/layouts/post.html
+breadcrumbs:
+  - {name: "2-run-ipfs-on-a-vps", link: "./" }
+id: 2-run-ipfs-on-a-vps
+date: 2015-11-02
+title: Run IPFS latest on a VPS
+author: Kyle Drake
+collection: posts
+---
+
+The best way to provide content using [IPFS](https://ipfs.io) is to run your own IPFS node. You can do this by running an IPFS node on your personal computer, but that will only work as long as your computer is running. For hosting content with a higher availability, you need to run an IPFS node in a datacenter. That ensures your content is always online and available to other nodes on the network.
+
+VPS instances provided by [Digital Ocean](https://www.digitalocean.com/), [Ramnode](http://ramnode.com/), [Linode](https://www.linode.com/), [Vultr](https://www.vultr.com/) and many other providers allow you to quickly setup your own Linux server with the reliability of a managed dedicated server without the full cost. This is a quick guide to setting up your own dedicated IPFS node on a VPS. We'll be using [Ubuntu](http://www.ubuntu.com/) 14.04LTS 64-bit for the example.
+
+First, some initial housekeeping I like to do. We're going to update the installed system to make sure that it has the latest updates. We're also going to turn on unattended updates, so that security patches can be done automatically without your intervention, and setup OpenNTP so the system clock is automatically updated:
```

Comment:
I don't think we should be telling users to do things like turning on unattended updates. This is a security policy with implications, and we shouldn't be setting it for them in this type of article. Yes, it is indeed helpful for new users of systems, but if this starts from "already having ubuntu installed", I think this would belong more in a "let's setup linux for your ipfs node" article, which might have more info, too, like which image to select from digital ocean, etc. (that might be a good post to have, like a "this is the VPS recommended for running ipfs nodes in the cloud, steps to do that). or if we want _this_ to be that article too (not just general vps, we could have two sections: (a) Setting up the VPS, and (b) Installing + running ipfs on it. ??